### PR TITLE
fix some typo under doc/

### PR DESCRIPTION
**Pull request by digger-yu**
fix some typo under doc/

detail info
        modified:   doc/analyticsBlog.asciidoc
        modified:   doc/packet_builder_yaml.asciidoc
        modified:   doc/release_notes_old.asciidoc
        modified:   doc/trex_appendix_asa_5585.asciidoc
        modified:   doc/trex_appendix_firmware_xl710.asciidoc
        modified:   doc/trex_appendix_linux_vf_config.asciidoc
        modified:   doc/trex_appendix_mellanox.asciidoc
        modified:   doc/trex_astf.asciidoc
        modified:   doc/trex_astf_vs_nginx.asciidoc
        modified:   doc/trex_book.asciidoc
        modified:   doc/trex_book_basic.asciidoc

### CODE REVIEW

File: doc/trex_astf.asciidoc [hunk 1]
```diff
@@ -1691,7 +1691,7 @@ Have a look at `astf/htttp_eflow2.py` example.
 See link:cp_astf_docs/api/profile_code.html#astfprogram-class[astf-program] for more info.
 
 ==== Tutorial: L7 emulation - Elephant flows with non-blocking send with tick var
-Same as the previous example, only instead of using `loop count` we are using time as a measurement. In the prevous example we calculated the received bytes in advance and use the rcv command with the right bytes values. However, sending & receiving data according to time is tricky and errors/dtops might occur (see stats from runnnig example below).
+Same as the previous example, only instead of using `loop count` we are using time as a measurement. In the prevous example we calculated the received bytes in advance and use the rcv command with the right bytes values. However, sending & receiving data according to time is tricky and errors/dtops might occur (see stats from running example below).
 
 Have a look at `astf/htttp_eflow4.py` example.
```
Comment (bjosv) [inline doc/trex_astf.asciidoc:1696]: There seems to be one more
```suggestion
Have a look at `astf/http_eflow4.py` example.
```

**Review by bjosv — COMMENTED** [review]
Nice!

File: doc/trex_astf.asciidoc (hunk 1 above)
Comment (digger-yu) [reply doc/trex_astf.asciidoc]: Added done.

**Review by bjosv — APPROVED** [review]

**Comment by hhaim** [comment]
rebased

**Pull request closed without merging**